lib/ios/AirMaps/AIRMapUrlTile.m

```diff
@@ -39,6 +39,13 @@ - (void) update
     if (_map == nil) return;
     [_map removeOverlay:self];
     [_map addOverlay:self level:MKOverlayLevelAboveLabels];
+    for (id<MKOverlay> overlay in _map.overlays) {
```

Comment:
Won't this have performance issues. Isn't there any way to decide the "zIndex" of an overlay so we don't have to remove and add the other overlays?

Reply:
I have tried insertOverlay with index equal to 0 but still not working.

Reply:
I have tried to use insertOverlay with atIndex equals 0. But still not working. So this is the solution I came up so far.

Reply:
I had a look and it seems that it's done the same way for the polylines and the rest of the views. Thank you.